### ISSUE TITLE
As OFA tech lead, I need the dependabot PR a11y review process documented and config file updated

### ISSUE THREAD

**Issue by ADPennington** [raft-tech/TANF-app#1449]
Currently, a11y is not part of our [git workflow](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Architecture%20Decision%20Record/009-git-workflow.md) for reviewing dependency-related PRs opened by Dependabot. Instead, we focus on reviewing the changelog to better understand how the upgrade will impact TDP (if at all), confirm that the dependency is actually still needed, and we test for breaking changes. 

However, a11y should still be considered when reviewing these upgrades to help ensure there are no impacts on TDP end users. During 11/17 dev sync, the team agreed to the following workflow process re: dependency PRs and a11y review:
- Update dependabot config to add `a11y review` label to frontend dependency PRs
- When dev picks up these tickets for review and testing, the release notes/changelog section should be reviewed to inspect for changes that could have a11y implications (user-facing changes)
- After testing, dev will follow existing frontend git workflow steps, which include tagging Miles for a11y review prior to moving the PR into QASP review. 

We recently decided to insert this process into the dependabot and git workflow ADRs, which are being updated as part of #1488.  We also need to update the config file to trigger reminder (via label) to the team to check for a11y-relevant changes to user-facing frontend dependencies. 

ACs:
- [x] Dependabot and Git workflow ADRs updated with PR review process re: a11y (#1488)
- [ ] Dependabot [config file](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/.github/dependabot.yml) updated to add `a11y review` label to frontend PRs. 

Tasks:
- [x] Dependabot and git-workflow ADRs updated [`ADPennington` ]
- [ ] update dependabot config file to add label to frontend tickets [`dev`]

Open Questions:
- [ ] Since the backend leverages a django fork dependency, under what conditions (if any) would we need a11y review as part of updates to that dependency (or the original django dependency? [@jorgegonzalez]

**Comment by ADPennington** [comment]
@valcollignon i've updated the stub of this ticket. Note:
- the review process is now reflected in the ADR updates
- this ticket can be closed once the dependabot config file update is addressed.

**Comment by abottoms-coder** [comment]
@ADPennington Does the open question refer to [this 508](https://github.com/raft-tech/django-admin-508) dependency?

**Comment by ADPennington** [comment]
> @ADPennington Does the open question refer to [this 508](https://github.com/raft-tech/django-admin-508) dependency?

just wondering if we need to keep an eye on a11y stuff when django is updated (so not just frontend dependencies). if so, then do we need to add an a11y label when django PRs are opened by dependabot? @abottoms-coder @jorgegonzalez
